bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/categorylist/FeaturedCategoryListImpl.java

```diff
@@ -106,6 +108,8 @@ private void initModel() {
         // After the identifier type has been determined, the specific list will be used further
         List<String> categoryIdentifiers = new ArrayList<>();
         assetOverride = new HashMap<>();
+        String categoryIdType = null;
+        String categoryFilterType;
```

Comment:
Could you check if you need this categoryFilterType variable? It looks like it's serving basically the same purpose as categoryIdType. If so, please keep categoryIdType only.